lib/Db/NewsMapperV2.php

```diff
@@ -56,9 +56,11 @@ public function __construct(
         $this->time = $time;
     }
 
-    public function update(Entity $entity): Entity
+    public function update(Entity $entity, bool $touch = true): Entity
     {
-        $entity->setLastModified($this->time->getMicroTime());
+        if ($touch === true) {
```

Comment:
This should use the `hasFieldsUpdated` or similar function instead of manually passing it.

Reply:
Noted, I'll try that

Reply:
@SMillerDev I've replaced the "manual touch" with `getUpdatedFields()` and `resetUpdatedFields()`.
Does it look okay for you?